httpgrpc/protocol_versions.go

```diff
@@ -1,5 +1,11 @@
 package httpgrpc
 
+import (
+	"google.golang.org/grpc/encoding"
+	grpcproto "google.golang.org/grpc/encoding/proto"
+	"strings"
```

Comment:
nit: need to manually re-group imports

Reply:
How do you generally do that in this repo?  I ran gofmt and make ci. :/